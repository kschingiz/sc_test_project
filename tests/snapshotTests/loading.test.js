import React from "react";
import LoadingBar from "../../components/loadingBar";
import renderer from "react-test-renderer";

it("LoadingBar renders correctly", () => {
  const tree = renderer.create(<LoadingBar />).toJSON();
  expect(tree).toMatchSnapshot();
});
