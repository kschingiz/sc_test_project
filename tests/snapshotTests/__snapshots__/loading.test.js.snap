// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingBar renders correctly 1`] = `
<div
  className="makeStyles-loadingScreen-1"
>
  <div
    className="css-0"
  >
    <div
      className="css-1a91y0u"
    />
    <div
      className="css-19gtrp6"
    />
    <div
      className="css-3vuga"
    />
    <div
      className="css-tbu3sb"
    />
    <div
      className="css-se1ybd"
    />
  </div>
</div>
`;
